db/migrate/20170605204001_create_chembl_molecules.rb

```diff
@@ -0,0 +1,40 @@
+class CreateChemblMolecules < ActiveRecord::Migration
+  def change
+    create_table :chembl_molecules do |t|
+      t.integer :molregno
+      t.string :pref_name, :limit => 255
```

Comment:
That's neat, I didn't know that. I made these specifications based on the existing specs from the ChEMBLdb, but if I ever have to adjust these (or create new such fields in the future), I'll just stick with `text`. Thanks!